dependencies/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+
+include(FetchContent)
+
+FetchContent_Declare(sqlpp1
+    GIT_REPOSITORY  https://github.com/rbock/sqlpp11
+    GIT_TAG         origin/develop
+)
+
+FetchContent_Declare(date
+    GIT_REPOSITORY https://github.com/HowardHinnant/date
+    GIT_TAG        v3.0.0
+)
+
+FetchContent_GetProperties(sqlpp1)
+if (NOT sqlpp1_POPULATED)
+    FetchContent_Populate(sqlpp1)
+    add_subdirectory(${sqlpp1_SOURCE_DIR} ${sqlpp1_BINARY_DIR})
+endif()
+
+FetchContent_GetProperties(date)
+if (NOT date_POPULATED)
+    FetchContent_Populate(date)
+    add_subdirectory(${date_SOURCE_DIR} ${date_BINARY_DIR})
+endif()
```

Comment:
There is no need to also fetch date here. Date is already a public dependency of sqlpp11